agi_plz_ortschaften_pub/build.gradle

```diff
@@ -53,15 +66,42 @@ task dbImport(type: Ili2pgImport, dependsOn: 'unzipData'){
     strokeArcs = true
 }
 
+task convertDataEdit(type: SqlExecutor, dependsOn: 'dbImportSogis'){
+    description = "Convert plz_ortschaften-Data to Edit Schema"
```

Comment:
Nein, hier wird ins Publikationsschema (welches allerdings in der edit-Datenbank liegt) transformiert.